src/transformers/utils/import_utils.py

```diff
@@ -842,11 +842,7 @@ def is_bitsandbytes_available():
     if not is_torch_available():
         return False
 
-    # bitsandbytes throws an error if cuda is not available
-    # let's avoid that by adding a simple check
-    import torch
```

Comment:
@Titus-von-Koeller

Reply:
As of bitsandbytes 0.43.1 it can be imported without CUDA being present (see merging commit https://github.com/bitsandbytes-foundation/bitsandbytes/commit/e2db55eda3f806b21eca595a790b80a2a60ab9ab and my PR implementing this, #1041).

This should probably have a simple version check; with bitsandbyteses older than that, `torch.cuda.is_available()` would be an additional requirement.